### PR TITLE
UT-Alignment-12d-5

### PR DESCRIPTION
**Pull request by LeeGregory12d**
## Unit test

alignment with left and right arcs and left and right leading and trailing clothoid transitions in horizontal and no vertical

## Covered topics

*tick one/many*

- [x] project setup
- [ ] georeferencing
- [x] alignment semantics
- [ ] alignment geometry
- [ ] linear placement
- [ ] project breakdown structure / spatial structure
- [ ] terrain
- [ ] triangulated irregular network
- [ ] open cross profile
- [ ] sectioned surface & solid
- [ ] pavement & course
- [ ] surface feature
- [ ] earthworks
- [ ] geotechnics
- [ ] longitudinal products (barriers, guardrail, ...)
- [ ] interchange
- [ ] structural
- [ ] drainage
- [ ] signage
- [ ] road furniture
- [ ] rail furniture
- [ ] ports & waterways furniture
- [ ] bridge furniture

### CODE REVIEW

File: UT-Alignment-12d-5/readme.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+ 
+### Intent
+
+IFC 4x3 RC2
+This scenario has an alignment with left and right arcs and left and right leading and trailing clothoid transitions in horizontal
+ and no vertical
```
Comment (pjanck) [inline UT-Alignment-12d-5/readme.md]:
```suggestion
 and no vertical alignment.
```

File: UT-Alignment-12d-5/readme.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- left clothoid transition of length 20 with IFC positive radius of curvature going from 0 to 150
+- left arc of length 74.118 with IFC positive radius of curvature
+- left clothoid transition of length 30 with IFC positive radius of curvature going from 150 to 0,
+- straight of length 14.9
+- left arc of length 12.45 with IFC positive radius of curvature 40
+- straight of length 47.26,
+- leading right clothoid transition of length 10 with IFC negative radius of curvature going from 0 to -75
+- right arc of length 26.798 with IFC negative radius of curvature -75,
+- trailing right clothoid transition of length 30 with IFC negative radius of curvature going from -75 to 0
+- straight of length 54.426
```
Comment (pjanck) [inline UT-Alignment-12d-5/readme.md]: Sometimes there's a comma at the end, sometimes there isn't - I can't quite figure out the pattern.

Also: perhaps a table would have been a better choice here?

**Review by pjanck — CHANGES_REQUESTED** [review]
Small readme improvements.

**Review by pjanck — APPROVED** [review]
LFTM

**Comment by pjanck** [comment]
Following the changes to the schema, (we are now at `RC3`), we ask you to update your IFC files accordingly. The schema can be found [here](https://github.com/bSI-InfraRoom/IFC-Specification/blob/main/IFC4x3_RC3.exp). The checking procedure for IFC files has been updated - please merge in the current state of the `main` branch.

**Review by jmirtsch — APPROVED** [review]

**Pull request merged**